### PR TITLE
Update 13-hosting.md

### PR DESCRIPTION
**Pull request by LibrEars**
add the non-profit association https://codeberg.org/ as public hosting service

<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
checkout@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

**Comment by scottgruber** [comment]
Hi @LibrEars 

I'm a new maintainer of this repository and reviewing pull requests and issues. The focus of Git for Novices is to focus on the fundamentals of Git and its unique lexicon. In the spirit of avoiding confusion by introducing a new host service I will close this PR. 

Thank you for taking the time to contribute.

**Comment by davidak** [comment]
To make the text less confusing for novice users, you can remove GitLab and BitBucket. Github as popular for-profit host and Codeberg as popular non-profit host should be enough choices.

**Comment by kekoziar** [comment]
> To make the text less confusing for novice users, you can remove GitLab and BitBucket. Github as popular for-profit host and Codeberg as popular non-profit host should be enough choices.

I respectfully don't agree. There are too many instances where people confuse git with GitHub. And, as the lesson is currently written - with use on GitHub as a platform for examples - there are accusations that it is just advertising for GitHub from some of those who do know the difference.  Removing references to GitLab and Bitbucket would reinforce both of these issues.

**Comment by kekoziar** [comment]
> Hi @LibrEars
> 
> I'm a new maintainer of this repository and reviewing pull requests and issues. The focus of Git for Novices is to focus on the fundamentals of Git and its unique lexicon. In the spirit of avoiding confusion by introducing a new host service I will close this PR.
> 
> Thank you for taking the time to contribute.

I am an existing maintainer, and agree with @scottgruber's assessment.  Codeberg is too new.  It's nonprofit status was only approved this year.  I hope they update and grow their organization, and look forward to being able to add this new hosting service when it is a more established hosting service.

**Comment by davidak** [comment]
@kekoziar thanks for your response.

When would you consider Codeberg established? It uses Gitea, which is established, and has many members, so i don't expect it to disappear anytime soon.

**Comment by kekoziar** [comment]
@davidak I would recommend waiting at least a year.  Here is my reasoning:  About 50% of non-profits fail or stall within the first few years. I'm not concerned with the reliability of Gitea. Non-profits fail for many other reasons, including financial and organizational.  

- The organization seems to have a good structure, and I highly support its mission, but I don't know how many people are actually supporting it.  As one example, the bylaws haven't been updated and formalized to indicate more permanency of the operations.  
- They only recently became an employer (June 2022)
- There are a lot of users, but very few members.  Reading the 200+ e.V.-members in whole-hundreds, that means there are less than 300 paying members of Codeberg. As of June 2022, only 138 were members with active voting rights, meaning there's a minimum income of only 3312€. Granted, I don't know what the actual income, but they also aren't explicit on where operational expenses are coming from (grants, government, only donations, etc). 
 
In summary, Codeberg is a very new organization, and will likely experience some growing pains. I hope it is able to navigate them successfully.  I think one year is a good time to reexamine including it in the curriculum.

I also have no problem bringing in @tobyhodges and other people from the Carpentries Core team for their input.

**Comment by davidak** [comment]
I respect your decision. Thanks for sharing your reasoning!

**Comment by aral** [comment]
@kekoziar Your point is entirely valid. However, there is a flip side to this, which is that we must support those organisations that we want to see become successful. If git-novice can play a role in _helping_ a not-for-profit like Codeberg become successful by making more people aware of it, I feel it would be in its ethical remit to do so.

_Disclaimer: Both me personally and Small Technology Foundation as an organisation support Codeberg with a €100 yearly donation each. It’s my/our way of trying to support what we want to see in the world._

**Comment by tobyhodges** [comment]
Thanks for tagging me in @kekoziar. Referring to [the consultation rubric](https://docs.carpentries.org/topic_folders/lesson_development/cac-consult-rubric.html), I believe a decision on this kind of change is for the lesson Maintainers to make and I trust you to look after the ongoing health of the lesson. I appreciate the thoughtful approach you and @scottgruber have taken when responding to this contribution.

**Comment by tobyhodges** [comment]
@aral & @davidak, you may find it ironic to do so in a space hosted on GitHub, but you would be welcome to develop an equivalent lesson teaching Git with Codeberg as the host for remote repositories in [The Carpentries Incubator](https://github.com/carpentries-incubator/proposals), if you would like to. I suspect it would be of interest to many members of The Carpentries community.

Of course, the lesson template and the git-novice lesson themselves are also open source, so you could also clone and develop that equivalent hosted on Codeberg if you prefer. But we could not offer any support from the Carpentries team in that case.
